terraform/README.md

```diff
@@ -0,0 +1,46 @@
+ScyllaDB Cloud has a [Terraform provider](https://github.com/scylladb/terraform-provider-scylladbcloud) which means that you can spin up new
+ScyllaDB Cloud clusters easily using Terraform. Follow the instructions below to
+set up the care-pet sample application in a ScyllaDB Cloud environment using Terraform.
+
+You'll set up Terraform to:
+1. Create a new ScyllaDB Cloud cluster (you need a [ScyllaDB Cloud account](https://cloud.scylladb.com/account/sign-up))
+1. Execute a CQL file that creates a new keyspace and tables for the care-pet project
+
+
+## Get started
+Go the the `terraform` directory and run `terraform init`
+```bash
+cd terraform/
+terraform init
```

Comment:
I'm getting the following 

````
15:27 $ terraform init

Initializing the backend...

Initializing provider plugins...
- Finding latest version of hashicorp/http...
- Finding latest version of scylladb/scylladbcloud...
- Finding latest version of hashicorp/null...
╷
│ Warning: Provider development overrides are in effect
│ 
│ The following provider development overrides are set in the CLI configuration:
│  - scylladb/scylladbcloud in /home/tzach/bin
│ 
│ Skip terraform init when using provider development overrides. It is not necessary and may error unexpectedly.
╵

╷
│ Error: Failed to query available provider packages
│ 
│ Could not retrieve the list of available versions for provider hashicorp/http: no available releases match the given constraints 
╵

╷
│ Error: Failed to query available provider packages
│ 
│ Could not retrieve the list of available versions for provider scylladb/scylladbcloud: no available releases match the given constraints 
╵

╷
│ Error: Failed to query available provider packages
│ 
│ Could not retrieve the list of available versions for provider hashicorp/null: no available releases match the given constraints 
```

Reply:
What if you skip running `terraform init` and just run `terraform apply`. If it still doesn't work it might be related to this issue https://github.com/hashicorp/terraform/issues/33097